### ISSUE TITLE
Warning: parse_option(): unknown option '--kerneltz'

### ISSUE THREAD

**Issue by zdenop** [k-szuster/luci-access-control#16]
I found out that rules created by this module are not working because of this line:
`option extra '--kerneltz'`
When I remove them, rules are working.

**Comment by zdenop** [comment]
One more info: 
when I restart firewall manually (/etc/init.d/firewall restart) I there better explanation:
```
   * Rule 'Phone'
     ! Exception: time: option "--kerneltz" can only be used once.
     ! Skipping due to previous exception (code 2)
   * Forward 'lan' -> 'wan'
```

**Comment by k-szuster** [comment]
There is a new release available, which corrects these errors: 0.4.2. Please upgrade.

**Comment by zdenop** [comment]
thanks.
